pyvista/plotting/mapper.py

```diff
@@ -224,7 +224,7 @@ def set_scalars(
                 rgb or _custom_opac,
             )
 
-            return show_scalar_bar, n_colors, clim
+            return show_scalar_bar, n_colors, clim, new_scalars
```

Comment:
Is this addition used anywhere? And should we document the return values now? Or is this an internal helper function?